source/whats-new.txt

```diff
@@ -26,6 +27,52 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+The 4.11 {+driver-short+} release includes **recursive schema support**
+and provides type safety for dot notation queries up to a depth of
+nine. Beyond a depth of nine, your code still compiles but TypeScript no longer
+type checks and values fall back to a type of ``any``. The recursive type
+depth limit is a current limitation in TypeScript.
+
+Feature Demonstration
+~~~~~~~~~~~~~~~~~~~~~
+
+Suppose we have a collection of type ``Collection<CircularSchema>``:
+
+.. code-block:: js
+   :copyable: false
+
+   interface CircularSchema {
+       name: string;
+       ref: CircularSchema;
+   }
+
+TypeScript enforces type checking below a depth of nine. The following
+code causes a TypeScript compilation error because the ``name`` property
+value must be a ``string`` type:
+
+.. code-block:: js
+   :copyable: false
+
+   collection.findOne({ 'ref.ref.ref.name': 25 })
+
+At a depth greater than nine, TypeScript compiles your code but no
+longer type checks it. The type falls back to type ``any``. The
```

Comment:
I think this version might be a little more clear and avoid the passive voice in the second sentence.

```suggestion
At a depth greater than nine, TypeScript compiles your code, but 
falls back to accepting the ``any`` type at that level. The
```

Reply:
I think the property type falls back to `any`, so I reworded to reflect that